graphql/schema/types/studio.graphql

```diff
@@ -29,6 +29,7 @@ input StudioCreateInput {
   name: String!
   url: String
   parent_id: ID
+  parent: StudioCreateInput
```

Comment:
Looking at the way this is processed in the resolver logic, I don't think it's a good design to include this. The behaviour isn't clear from the outside and the behaviour is based on whether an existing studio exists or not. The client should be responsible for creating/updating the parent studio. The only benefit I see in combining these operations is that you end up with a single transaction to roll back if needed, but I don't think it's worth it.

Reply:
I've moved all parent studio logic out of the `sqlite` package.
I think I was trying to treat it logically as a single operation (not necessarily a single transaction). I wanted to avoid duplicating all of the parent matching/creating/updating logic in all of the multiple places where studios are created (identify/import/batch update/tagger view).
All of that code duplication can lead to bugs in the future when there's a new way to create a studio but all of the parent logic is not copied exactly. Like when disambiguation was added, it was added to all of the various places you can create a performer, but missed in identify.

I think this comment means you're suggesting to also write this parent logic in the frontend, so that the frontend can make an api call to create/update a parent studio, then attach it to the main studio and make another api call to create/update studio?